modules/objects.js

```diff
@@ -0,0 +1,23 @@
+const tasks = [
+  {
+    string: 'Complete to do list project',
+    boo1: 'Join stand-up Call',
+    id: '1',
+  },
+  {
+    string: 'Join stand-up Call',
+    boo1: 'Not completed',
+    id: '2',
+  },
+  {
+    string: 'Arange a call',
+    boo1: 'Join stand-up Call',
+    id: '3',
+  },
+  {
+    string: 'Control the teams project',
+    boo1: 'Join stand-up Call',
+    id: '4',
+  },
+];
```

Comment:
- [ ] According to the requirement, you need to set an array of some simple to-do tasks (array of objects). Each task object should contain three keys. that is: `description [`string`], completed [`bool`], index: [`number`]`. Please make it like this.
`const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
];
`